experimental/apps-mcp/lib/templates/appkit/databricks.yml.tmpl

```diff
@@ -0,0 +1,47 @@
+bundle:
```

Comment:
You seem to have two files here: `databricks.yml` + `databricks.yml.tmpl`

Reply:
Having the two databricks.yml files is intentional. I had the logic such that it would always prefer the tmpl and having both allowed me to test the template inline. I have since changed the template to be a proper bundle template and had to remove the non-tmpl file